SSL handshake in Jenkins

Running the decops-cm-client in jenkins, we're facing connection issues with SSL handshake

When running a trial-command in jenkins, we cann't establish the connection:
`"+ mkdir -p /var/apphome/xxxxx/.jenkins/workspace/CI_Fiori_ABAPBackend_master_Transportauftrag/tmp"
"+ cp -R /var/apphome/<...>/dist.cli/ /var/apphome/xxxxx/.jenkins/workspace/CI_Fiori_ABAPBackend_master_Transportauftrag/tmp/dist.cli/"
"+ /var/apphome/xxxxx/.jenkins/workspace/CI_Fiori_ABAPBackend_master_Transportauftrag/tmp/dist.cli/bin/cmclient -t SOLMAN -e https://xxxxxx/sap/opu/odata/sap/AI_CRM_GW_CM_CI_SRV/ -u xxxx -p [xxxxx] is-change-in-development -cID xxxxxxx879"

"Exception in thread "main" org.apache.olingo.client.api.http.HttpClientException: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
                   at org.apache.olingo.client.core.communication.request.AbstractODataRequest.doExecute(AbstractODataRequest.java:303)
                   at org.apache.olingo.client.core.communication.request.retrieve.ODataEntityRequestImpl.execute(ODataEntityRequestImpl.java:57)
                   at org.apache.olingo.client.core.communication.request.retrieve.ODataEntityRequestImpl.execute(ODataEntityRequestImpl.java:37)
                   at sap.ai.st.cm.plugins.ciintegration.odataclient.CMODataSolmanClient.getChange(CMODataSolmanClient.java:101)
                   at sap.prd.cmintegration.cli.GetChangeStatus.execute(GetChangeStatus.java:60)
                   at sap.prd.cmintegration.cli.GetChangeStatus.main(GetChangeStatus.java:91)
                   at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
                   at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
                   at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
                   at java.lang.reflect.Method.invoke(Method.java:498)
                   at sap.prd.cmintegration.cli.Commands.main(Commands.java:312)
                   at sap.prd.cmintegration.cli.ExitWrapper.main(ExitWrapper.java:15)
Caused by: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target"
`_

Running the same command directly with users cli, the connection works. 
We export the variables before the call through JAVA_OPTS:

`export 'JAVA_OPTS= -Djdk.tls.ephemeralDHKeySize=2048 -Djava.protocol.handler.pkgs=org.apache.catalina.webresources -Dorg.apache.catalina.security.SecurityListener.UMASK=0027 -Djavax.net.ssl.trustStore=/var/apphome/xxxx/conf/keystore/trustStore.jks -Djavax.net.ssl.trustStorePassword=…‘`

Question: what does the devops_cm_client expect? is it possible to adapt the call in a way to directly set the truststore or set the loglevel to debug?

Thanks

--- a/README.md
+++ b/README.md
@@ -49,6 +49,8 @@ This command line client can be consumed either as a Java application from [mave
 <CMD> [COMMON_OPTIONS...] <subcommand> [SUBCOMMAND_OPTIONS] <parameters...>
 ````
 
+To pass additional Java options to the command e.g. another truststore you can set the environment variable `CMCLIENT_OPTS`
+
 | Option                   |     Description         |
 |--------------------------|-------------------------|
 | `-t`, `--backend-type`   | `SOLMAN` or `CTS`       |

--- a/README.md
+++ b/README.md
@@ -49,7 +49,7 @@ This command line client can be consumed either as a Java application from [mave
 <CMD> [COMMON_OPTIONS...] <subcommand> [SUBCOMMAND_OPTIONS] <parameters...>
 ````
 
-To pass additional Java options to the command e.g. another truststore you can set the environment variable `CMCLIENT_OPTS`
+To pass additional Java options to the command (for example, another truststore), set the environment variable `CMCLIENT_OPTS`
 
 | Option                   |     Description         |
 |--------------------------|-------------------------|

